cabal-docspec/src/CabalDocspec/GHCi.hs

```diff
@@ -104,10 +105,10 @@ waitGhci _tracer (GHCi iph mt) mitVar microsecs = do
 
     -- send separator
     separator <- show <$> genString
-    Proci.sendTo iph $ fromString $ separator ++ " :: " ++ mt' ++ "\n"
+    Proci.sendTo iph $ fromString $ separator ++ " :: " ++ mt' ++ newlineStr
```

Comment:
Have you tested that `\r\n` doesn't cause any weirdness when *sent to*?

Reply:
I can successfully run the doctests for `comonad` and `lens` with this patch. Is there some other way you'd like for me to test this?

Reply:
That's fine with me.